lib/js_of_ocaml/import.ml

```diff
@@ -35,21 +35,21 @@ module Poly = struct
 end
 
 module Int_replace_polymorphic_compare = struct
-  let ( < ) (x : int) y = x < y
+  external ( < ) : int -> int -> bool = "%lessthan"
```

Comment:
Why is this change in the PR?

Reply:
I've added the commit while working on the PR as it generate nicer code. But it doesn't have to be part of this PR.